src/Business Foundation/Test/NoSeries/src/ERMNoSeriesTests.Codeunit.al

```diff
@@ -0,0 +1,604 @@
+namespace Microsoft.Test.Foundation.NoSeries;
+
+using System.TestLibraries.Utilities;
+using System.TestLibraries.Security.AccessControl;
+using Microsoft.Foundation.NoSeries;
+
+codeunit 134370 "ERM No. Series Tests"
```

Comment:
CodeSpell checker shows same typo's in this codeunit

![image](https://github.com/microsoft/BCApps/assets/25268332/6e309e6e-cc19-4171-8e90-ccbc90afcffc)